sheets/api.py

```diff
@@ -1012,7 +1045,8 @@ def process_assignment_spreadsheets(self):
                 log.error(
                     "All relevant coupons have been assigned and messages have been sent, "
                     "but failed to update the spreadsheet properties to indicate status "
-                    "- %s, exception: %s" % (spreadsheet_repr(spreadsheet), str(exc))
+                    "- %s, exception: %s",
+                    (spreadsheet_repr(spreadsheet), str(exc)),
```

Comment:
Should this be `log.exception` instead? I think `str(exc)` will only get the exception message, not the stack trace

Reply:
For a solid few days working on this, I forgot that `log.exception` existed 😬